apps/ejabberd/src/mod_offline.erl

```diff
@@ -376,17 +371,14 @@ inspect_packet(Acc, From, To, Packet) ->
             Acc
     end.
 
-store_packet(
-        From,
-        To = #jid{luser = LUser, lserver = LServer},
-        Packet = #xmlel{children = Els}) ->
+store_packet(From, To = #jid{luser = LUser, lserver = LServer},
+             Packet = #xmlel{children = Els}) ->
     TimeStamp =
-    case xml:get_subtag(Packet, <<"delay">>) of
-        false ->
+    case exml_query:subelement(Packet, <<"delay">>) of
+        undefined ->
             now();
-
-        #xmlel{name= <<"delay">>, attrs=Attr} ->
-            case xml:get_attr_s(<<"stamp">>, Attr) of
+        #xmlel{name= <<"delay">>, attrs=Attr} = DelayEl ->
```

Comment:
I know you have plans to fix all the compilers complains re `now` in it's own dedicated PR. Here compiler noticed an unused var. Could you fix this?